department-of-veterans-affairs/va.gov-cms
# VACMS-19185: Adding Manila menu & enabling it on relevant content types

## Description

Relates to #19185

## Testing done

manual in tugboat
- [x] Content > Menus > All Menus > [Manila VA Clinic](https://pr19193-xpmqgexjxlwcomjru2bcrenvqirlz1pp.ci.cms.va.gov/admin/structure/menu/manage/manila-va-clinic) 
- [x] event
- [x] event_listing
- [x] VAMC facility
- [x] VAMC system
- [x] health service listing
- [x] leadership listing
- [x] locations listing
- [x] news release
- [x] news release listing
- [x] publication listing
- [x] staff profile
- [x] story
- [x] story listing
- [x] VAMC operating status
- [x] VAMC billing and insurance
- [x] VAMC medical records office
- [x] VAMC system policies page
- [x] VAMC register for care
- [x] VAMC VA police
- [x] VAMC detail page


## Screenshots

![Screenshot 2024-09-12 at 3 48 49 PM](https://github.com/user-attachments/assets/2cc52423-aabb-4cd9-be80-4529e2b6c2ff)

![screencapture-pr19193-xpmqgexjxlwcomjru2bcrenvqirlz1pp-ci-cms-va-gov-admin-structure-menu-manage-manila-va-clinic-2024-09-12-16_34_09](https://github.com/user-attachments/assets/5bd4d3d8-a125-48e4-8afd-76437f0677c0)

![Screenshot 2024-09-12 at 4 37 55 PM](https://github.com/user-attachments/assets/260c74bf-8990-4992-86fa-9449dd81f001)


## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As a Content Admin (e.g. david.pickett@va.gov)
1. Log into [the Tugboat](https://pr19193-xpmqgexjxlwcomjru2bcrenvqirlz1pp.ci.cms.va.gov/)
2. Go to Content > Menus > All Menus > [Manila VA Clinic](https://pr19193-xpmqgexjxlwcomjru2bcrenvqirlz1pp.ci.cms.va.gov/admin/structure/menu/manage/manila-va-clinic) 
   - [x] Confirm Manila VA Clinic menu exists
3. Check that `<Manila VA Clinic>` appears in the Menu settings / Parent link drop down menu for each of these content types
   - [x] Event
   - [x] Event_listing
   - [x] VAMC facility
   - [x] VAMC system
   - [x] health service listing
   - [x] leadership listing
   - [x] locations listing
   - [x] news story
   - [x] staff profile
   - [x] news release
   - [x] news release listing
   - [x] publication listing
   - [x] story listing
   - [x] VAMC operating status
   - [x] VAMC billing and insurance
   - [x] VAMC medical records office
   - [x] VAMC system policies page
   - [x] VAMC register for care
   - [x] VAMC VA police

### Definition of Done
